packages/viz/src/lib/style/default-styles.ts

```diff
@@ -1,53 +1,121 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// TODO remove any
```

Comment:
I would create a ticket for it removing the `TODO`, if not this line could be here for years :)

Reply:
In fact, I'm not sure if we want to remove this any. It's pretty straight forward and it's for an internal where we take advantage of the dynamic flexibility of JS. 

If we improve it, I'd like a way to explore how to do this without defining a strong interface